userspace/libsinsp/CMakeLists.txt

```diff
@@ -133,11 +133,12 @@ endif()
 
 if(NOT WIN32 AND NOT APPLE)
 list(APPEND SINSP_SOURCES
-	grpc_channel_registry.cpp
+	async_cgroup.cpp
 	cri.cpp
 	container_engine/cri.cpp
 	${CMAKE_CURRENT_BINARY_DIR}/cri.grpc.pb.cc
-	${CMAKE_CURRENT_BINARY_DIR}/cri.pb.cc)
+	${CMAKE_CURRENT_BINARY_DIR}/cri.pb.cc
+	grpc_channel_registry.cpp)
```

Comment:
Nit: Suggest you put the `)` on a newline, that way when we add new files it doesn't get marked as a change when we invariably have to move the paren.